kiwi/boot/image/builtin_kiwi.py

```diff
@@ -52,7 +52,7 @@ def prepare(self):
             root_dir=self.boot_root_directory,
             allow_existing=True
         )
-        manager = system.setup_repositories()
+        manager = system.setup_repositories(signing_keys=self.signing_keys)
```

Comment:
The same as before, setup_repositories also includes `clear_cache` with a default values in its signature, we need to clearly state this parameter goes for `signing_keys` instead of `clear_cache`.